obsah/constraints.py

```diff
@@ -21,8 +21,11 @@ def variable_to_parameter(name):
     errors = []
     for constraint in constraints.get('required_together', []):
         present_args = [arg in args for arg in constraint]
+        missing_args = [arg for arg in constraint if arg not in args]
```

Comment:
Perhaps only calculate this in the `if` since it's only needed there? Might be a micro-optimization, but still

Reply:
sure, why not